manifests/global.yaml

```diff
@@ -565,7 +565,7 @@ global:
 
   # Configure the policy for validating JWT.
   # Currently, two options are supported: "third-party-jwt" and "first-party-jwt".
-  jwtPolicy: "third-party-jwt"
+  jwtPolicy: "first-party-jwt"
```

Comment:
We should have 3p default i think. https://github.com/istio/istio/pull/20393 will make it so we can autodetect it in most cases, so the UX is fine.

Reply:
In this PR, it is set as "first-party-jwt" so this option gets tested in all unit/integration tests. I can create a follow up PR to set the default value.

Reply:
Ok, good idea